plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -86,13 +91,35 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 		glog.V(4).Info(msg)
 		return admission.NewForbidden(a, errors.New(msg))
 	}
-	if len(sc.Plans) > 1 {
+	// find all the service plans that belong to the service class
+
+	// TODO: Need to be careful here. Is it possible to have only one
+	// serviceplan available while others are still in progress?
+
+	// implement field selector
+
+	// loop over all service plans accumulate into slice
+	plans, err := d.spLister.List(labels.Everything())
+	// TODO filter `plans` down to only those owned by `sc`.
+
+	// check if there were any service plans
+	// TODO: in combination with not allowing classes with no plans, this should be impossible
+	if len(plans) <= 0 {
+		msg := fmt.Sprintf("no plans found at all")
```

Comment:
no variable args for the Sprintf, so just a strcpy would do.  However, it might be nice to include the service name in there.

Reply:
ya, sure I'll add the name.